src/api-client.ts

```diff
@@ -185,4 +196,36 @@ export class RevAiApiClient {
             }
         }
     }
+
+    async getCaptions(id: string, fileName = null, fileLocation = null): Promise<string> {
+        try {
+            const response = await axios.get(`/jobs/${id}/captions`, {
+                headers: { 'Accept': 'application/x-subrip' }
+            });
+
+            if (fileName) {
+                const fullPath = path.join(fileLocation || '', `${fileName}.srt`);
```

Comment:
Yes, I think it's premature to add save to file feature since we did not discuss it yet.